pkg/api/v1/testkube/model_variables_extended.go

```diff
@@ -20,3 +15,13 @@ func VariablesToMap(v Variables) map[string]string {
 
 	return vars
 }
+
+func ObfuscateSecrets(output string, variables Variables) string {
+	for _, v := range variables {
+		if v.Type_ == VariableTypeSecret {
+			output = strings.ReplaceAll(output, `'`+v.Value+`'`, "********")
+			output = strings.ReplaceAll(output, `"`+v.Value+`"`, "********")
```

Comment:
This is even more complex. We don't have values for secret vars with secret references. They are propagated when we run an executor pod

Reply:
We have Obfuscate logic already implemented

<img width="621" alt="image" src="https://user-images.githubusercontent.com/30776/188800220-6a0f479a-8f08-43b5-8baf-81cc85e2eb37.png">